### PR TITLE
Feature/209 fullscreen gallery

### PR DESCRIPTION
**Pull request by Mortadha-abderrhim**
Show Gallery in Fullscreen. It is now possible to scroll between images in fullscreen
Screenshot_1620331648

### CODE REVIEW

File: app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt [hunk 1]
```diff
@@ -157,24 +158,34 @@ class ExperienceActivity : AppCompatActivity() {
...
-        fullScreenLayout.addView(imageView)
-        fullScreenLayout.visibility = View.VISIBLE
-        fullScreenLayout.setOnClickListener {
-            (it as RelativeLayout).removeView(it[1])
-            it.visibility = View.GONE
-        }
+        supportFragmentManager.beginTransaction()
+            .remove(fragment)
+            .commit()
+        fullScreenLayout.visibility= View.GONE
```
Comment (aouame) [inline app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt:187]: Wrong spacing here.

File: app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt [hunk 2]
```diff
@@ -157,24 +158,34 @@ class ExperienceActivity : AppCompatActivity() {
...
+        val fragment = GalleryBlockFragment(isFullscreen = true)
+        fragment.arguments = bundleOf(
+            BlockFragmentArguments.GALLERY_BLOCK_IMAGE_URLS to imageURLs,
+            BlockFragmentArguments.GALLERY_BLOCK_FULLSCREEN_POSITION to selectedPosition
         )
+        supportFragmentManager.beginTransaction()
+            .add(R.id.fullScreenLayout, fragment)
+            .commit()
+
+        fullScreenLayout.visibility= View.VISIBLE
```
Comment (aouame) [inline app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt:174]: Wrong spacing here

File: app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt [hunk 3]
```diff
@@ -157,24 +158,34 @@ class ExperienceActivity : AppCompatActivity() {
...
+        startTourVisibility = startTourView.visibility
+        startTourView.visibility= View.GONE
```
Comment (aouame) [inline app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt:176]: Here too

File: app/src/main/java/com/pindex/main/utils/Constants.kt [hunk 4]
```diff
@@ -64,6 +64,10 @@ object Constants {
          * The Gallery block image descriptions argument
          */
         const val GALLERY_BLOCK_IMAGE_DESCRIPTIONS: String = "imageDescriptions"
+        /**
+         * The full screen Gallery block current Position
+         */
+        const val GALLERY_BLOCK_FULLSCREEN_POSITION : String="currentPosition"
```
Comment (aouame) [inline app/src/main/java/com/pindex/main/utils/Constants.kt:70]: Here

**Review by aouame — COMMENTED** [review]
You should reformat ExperienceActivity and Constants files. Other than that it looks good !

**Review by aouame — APPROVED** [review]

**Review by hadiyahya11 — APPROVED** [review]
Everything looks in order to me. Well done

**Pull request merged**